src/main/java/core/basesyntax/service/impl/ParserImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.TransactionDto;
+import core.basesyntax.service.Parser;
+import core.basesyntax.service.Validator;
+
+public class ParserImpl implements Parser {
```

Comment:
ты же сделала парсер дженериком 
```suggestion
public class ParserImpl implements Parser<TransactionDto> {
```